pkg/products/threescale/prometheusRules.go

```diff
@@ -275,6 +275,19 @@ func (r *Reconciler) newAlertReconciler() resources.AlertReconciler {
 						For:    "15m",
 						Labels: map[string]string{"severity": "warning"},
 					},
+					{
+						Alert: "ThreescaleApicastWorkerRestart",
+						Annotations: map[string]string{
+							"summary":     "A new worker process in Nginx has been started",
```

Comment:
@pepedocs  and @jjaferson  I notice the annotation for the alert mentions this `Annotations: map[string]string{
							"summary":     "A new worker process in Nginx has been started",` should this not review and describe the particular component in RHMI or RHOAM?

The link SOP would need improving also @jjaferson  for example: **https://github.com/3scale/3scale-Operations/blob/master/sops/alerts/apicast_worker_restart.adoc**

```
Pod hits the memory limits and one of the worker processes was killed. Check why the limits were reached, and if it’s needed, increase them

Node is out of memory, so try to kill containers to schedule to another node

```
We should have an example of the pod hitting it memory limit and what that would look like. What action can we take to patch, scale the update the pods memory via request limits or budget disruption budget.

`oc events` how to up the memory of the pod using a `oc patch` command etc.

What action can an CS-SRE perform on this component for this managed service?

Reply:
@jjaferson @Nanyte25 as discussed on the sync call with @brizrobbo, CS-SRE will take an action to work with 3Scale to improve the content of the SOP itself. That work doesn't block this PR from being merged as the SOP URL won't change.

Reply:
Sounds good to me @pmccarthy